packages/react-app/src/constants.ts

```diff
@@ -176,11 +176,3 @@ export const DEFAULT_FILTER = {
   bounty: undefined,
   status: ENUM_QUEST_STATE.Active,
 } as FilterModel;
-
-export const DEAULT_CLAIM_EXECUTION_DELAY_MS = 60 * 1000;
-
-export const DEFAULTS = {
-  DEFAULT_PAGE,
-  DEFAULT_FILTER,
-  DEAULT_CLAIM_EXECUTION_DELAY_MS,
-};
```

Comment:
@Corantin Its not been used, looks safe to remove, you can confirm?

Reply:
this one should be use :S DEAULT_CLAIM_EXECUTION_DELAY_MS

Reply:
In quest-service: 
![image](https://user-images.githubusercontent.com/13890618/162465272-3db72428-2fa8-431e-8d49-4667cfa9bb67.png)